spec/workers/publish_links_worker_spec.rb

```diff
@@ -8,8 +8,14 @@
 
     context "with valid links" do
       before do
-        allow_any_instance_of(BulkTagging::TagMapping).to receive(:valid?)
-          .and_return(true)
+        mapping_double = instance_double(
+          BulkTagging::TagMapping,
+          valid?: true,
+          errors: [],
+          mark_as_errored: nil,
+          mark_as_tagged: nil,
```

Comment:
Wondering why we adding `mark_as_errored` and ` mark_as_tagged` here?

Reply:
These are methods that the mock object needs in it's usage.